install_config/router/customized_haproxy_router.adoc

```diff
@@ -115,21 +115,24 @@ The *define* action names the file that will contain the processed template.
 [cols="2*", options="header"]
 |===
 |Function | Meaning
-|`*endpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit) []Endpoint*` | Returns the list of valid endpoints.
-|`*env(variable, default string) string*` | Tries to get the named environment variable from the pod. Returns the second
-argument if the variable cannot be read or is empty.
+|`*processEndpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit, action string) []Endpoint*` |
+Returns the list of valid endpoints. When action is "shuffle", the order of endpoints is randomized.
+|`*env(variable, default ...string) string*` | Tries to get the named environment variable from the pod.
```

Comment:
Here should be `*env(variable string, defaults ...string) string*`

Reply:
changed